CONTRIBUTING.rst

```diff
@@ -34,8 +34,10 @@ Next, ``cd`` to the pycodestyle repository that you cloned earlier and
 create, then activate a virtualenv::
 
     $ cd pycodestyle
-    $ virtualenv pycodestyle-venv
-    $ source pycodestyle-venv/bin/activate
+    $ virtualenv venv
+    $ source venv/bin/activate
+
+Note that ``venv/`` is ignored via ``.gitignore``.
```

Comment:
so I assume that the original author intended for the project name to be in the venv name so it would show up in the `PS1` -- I think we can make _both_ work with something this:

### in `.gitignore`

```
/venv*
```

### in docs

suggest `venv-pycodestyle`

(and it'll also work for the common workflow of a venv just named `venv` or `venv2` or `venv3`)

Reply:
Done. (Would still be nice to call out `gitignore`, 'cause that's the first thing I'll look at when generating auxiliary artifacts.)